### PR TITLE
Travis CI

### PR DESCRIPTION
**Pull request by iLtc**
This PR added Travis CI supports for our projects. It also fixes some bugs for previous test cases.

## Checklist:

- [x] I have attached this PR to an issue/user story

### CODE REVIEW

**Comment by gauridhawan** [comment]
I hate to say this but adding Travis and bug fixes should probably be in 2 different PRs since they are completely unrelated. Do you think you can split the two?

**Review by KHafner — APPROVED** [review]

**Comment by iLtc** [comment]
> I hate to say this but adding Travis and bug fixes should probably be in 2 different PRs since they are completely unrelated. Do you think you can split the two?

Okay, I just removed the bug fixing part. @KHafner will raise a new PR to fix them.

This PR will not pass the test now. But I think it is okay.

File: tests/test_models.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 from service import app
 
 DATABASE_URI = os.getenv("DATABASE_URI", "sqlite:///../db/test.db")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
```
Comment (gauridhawan) [inline tests/test_models.py:27]: Why have we set it to False? Just curious, I don't think it's a bug or something but wanted to know why we do this

**Review by gauridhawan — APPROVED** [review]
LGTM!

File: tests/test_models.py (hunk 1 above)
Comment (iLtc) [reply tests/test_models.py:27]: This is a historical issue. SQLAlchemy will print a lot of warning if we did not set it to False. 

We will create a config file later and move this line there.

**Pull request merged**